hc/accounts/migrations/0007_auto_20180503_0944.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Hey @kmunene Let's remove this migrations file from the PR. The rest of the team will make migrations once they have the updated repo content.